source/common/upstream/outlier_detection_impl.h

```diff
@@ -145,7 +147,14 @@ class DetectorHostMonitorImpl : public DetectorHostMonitor {
   GAUGE  (ejections_active)                                                                        \
   COUNTER(ejections_overflow)                                                                      \
   COUNTER(ejections_consecutive_5xx)                                                               \
-  COUNTER(ejections_success_rate)
+  COUNTER(ejections_success_rate)                                                                  \
+  COUNTER(ejections_enforced_total)                                                                \
```

Comment:
Can you add these to https://github.com/envoyproxy/data-plane-api/blob/320b5afbd0f6ed88df8d8e361545decabfea7777/docs/root/configuration/cluster_manager/cluster_stats.rst (and also indicate there what is deprecated and not)?

Reply:
Done in https://github.com/envoyproxy/data-plane-api/pull/245/